Source/Mosa.DeviceSystem/IFrameBuffer.cs

```diff
@@ -7,53 +7,38 @@ namespace Mosa.DeviceSystem
 	/// </summary>
 	public interface IFrameBuffer
 	{
-		/// <summary>
-		/// Gets the width.
-		/// </summary>
+		/// <summary>Gets the width.</summary>
 		/// <value>The width.</value>
-		uint Width { get; }
+		public uint Width { get; }
 
-		/// <summary>
-		/// Gets the height.
-		/// </summary>
+		/// <summary>Gets the height.</summary>
 		/// <value>The height.</value>
-		uint Height { get; }
+		public uint Height { get; }
 
-		/// <summary>
-		/// Gets the pixel.
-		/// </summary>
+		/// <summary>Gets the pixel.</summary>
 		/// <param name="x">The x.</param>
 		/// <param name="y">The y.</param>
-		/// <returns></returns>
-		uint GetPixel(uint x, uint y);
+		public uint GetPixel(uint x, uint y);
 
-		/// <summary>
-		/// Sets the pixel.
-		/// </summary>
+		/// <summary>Sets the pixel.</summary>
 		/// <param name="color">The color.</param>
 		/// <param name="x">The x.</param>
 		/// <param name="y">The y.</param>
-		void SetPixel(uint color, uint x, uint y);
+		public void SetPixel(uint color, uint x, uint y);
 
-		/// <summary>
-		/// Fills a rectangle with color.
-		/// </summary>
+		/// <summary>Fills a rectangle with color.</summary>
 		/// <param name="color">The color.</param>
 		/// <param name="x">X of the top left of the rectangle.</param>
 		/// <param name="y">Y of the top left of the rectangle.</param>
 		/// <param name="w">Width of the rectangle.</param>
 		/// <param name="h">Width of the rectangle.</param>
-		void FillRectangle(uint color, uint x, uint y, uint w, uint h);
+		public void FillRectangle(uint color, uint x, uint y, uint w, uint h);
 
-		/// <summary>
-		/// If using double buffering, copies the second buffer to the screen.
-		/// </summary>
-		void Update();
+		/// <summary>If using double buffering, copies the second buffer to first buffer (screen).s</summary>
```

Comment:
Comment looks like it has an issue

Reply:
It was a small typo in the comment section. Should be fixed now.